processor/src/main/java/com/linecorp/decaton/processor/runtime/ProcessorProperties.java

```diff
@@ -179,6 +180,22 @@ public class ProcessorProperties extends AbstractDecatonProperties {
             PropertyDefinition.define("decaton.deferred.complete.timeout.ms", Long.class, -1L,
                                       v -> v instanceof Long);
 
+    /**
+     * Timeout for destroying processors.
```

Comment:
I just realized the word `processor` is used in various context also for other properties...

To clarify the effect of this property, how about rewording to `Timeout for processor threads termination` ?

And also renaming the property to `decaton.processor.threads.termination.timeout.ms` or something